src/containers/search_bar.js

```diff
@@ -32,23 +63,32 @@ class SearchBar extends Component {
     }
     
     return searchedMovies.map(movie => {
-      return  <SearchedMovies key={movie.id} movie={movie} />
+      return  (
+      <SearchedMovies 
+        key={movie.id}
+        movie={movie} 
+      />
+    )
     });
   };
   
   render() {
+    const searchLoader = this.state.searchLoader;
     return (
-      <div>
+      <div ref={this.searchBar}>
+        <SearchLoader loading={searchLoader} />
```

Comment:
Render loader or not depending on this.state.searchLoader, no need to pass the props to the loader component.